internal/integration_test/vs/bench_allocation.go

```diff
@@ -51,13 +51,24 @@ func allocationCall(m Module, _ int) error {
 	}
 
 	// Now, we can call "greet", which reads the string we wrote to memory!
-	if err = m.CallI32I32_V(testCtx, "greet", namePtr, nameSize); err != nil {
-		return err
+	ptrSize, fnErr := m.CallI32I32_I64(testCtx, "greet", namePtr, nameSize)
+	if fnErr != nil {
+		return fnErr
 	}
 
 	// This pointer was allocated by Rust, but owned by Go, So, we have to
```

Comment:
Nit: TinyGo not Rust.

Reply:
yeah, that was part already of the code so I didn't change it

Reply:
leaving things better than you found them is a good practice! please fix typos as you see them.

Reply:
done